miner/algo_common.go

```diff
@@ -262,14 +262,31 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 	bundleSimEffGP := new(big.Int).Set(bundle.MevGasPrice)
 
 	// allow >-1% divergence
-	bundleActualEffGP.Mul(bundleActualEffGP, big.NewInt(100))
+	bundleActualEffGP.Mul(bundleActualEffGP, common.Big100)
 	bundleSimEffGP.Mul(bundleSimEffGP, big.NewInt(99))
 
 	if bundleSimEffGP.Cmp(bundleActualEffGP) == 1 {
 		log.Trace("Bundle underpays after inclusion", "bundle", bundle.OriginalBundle.Hash)
 		return errors.New("bundle underpays")
 	}
 
+	if enforceProfit {
+		// if profit is enforced between simulation and actual commit, only allow >-1% divergence
+		simulatedBundleProfit := new(big.Int).Set(bundle.TotalEth)
+		actualBundleGasFees := new(big.Int).Mul(bundleActualEffGP, big.NewInt(int64(gasUsed)))
+		actualBundleProfit := new(big.Int).Add(coinbaseBalanceDelta, actualBundleGasFees)
+
+		// We want to make simulated profit smaller to allow for some leeway in cases where the actual profit is
+		// lower due to transaction ordering
+		simulatedBundleProfit.Mul(simulatedBundleProfit, big.NewInt(99))
+		actualBundleProfit.Mul(actualBundleProfit, common.Big100)
+
+		if simulatedBundleProfit.Cmp(actualBundleProfit) > 0 {
```

Comment:
I think that the 99% cutoff is very aggressive for profit, especially that for regular txs you don't take into account unused gas

Reply:
99% was recommendation from @dvush 

From a spot check analysis it appears rare for low profit to occur

Reply:
I don't know what cutoff is better here. 5%?

Reply:
updated to 5%